index.d.ts

```diff
@@ -4,6 +4,10 @@ declare namespace firstRun {
 		The name used to identify it. Default: `name` field in your package.json
 		*/
 		readonly name?: string;
+		/**
+		The version used to identify it. Default: if `name` is not set, `version` field in your package.json, otherwise `undefined`.
```

Comment:
I don't understand this, why does it only use the `version` field when `name` is NOT set? Could be written clearer.

Reply:
Well, we can change that, but I suppose when people explicitly set name, they want it to be different from what is in `package.json`, so using version we can parse will be misleading(even if the name in `package.json` is the same as explicit name).

Reply:
Overall, the goal was that if someone just uses a oneliner without additional options, it can be upgraded to new functionality without any code changes, but doesn't introduce any unexpected things if someone used additional options before.